kernel/model/scope.go

```diff
@@ -19,15 +19,52 @@ package model
 import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"sort"
+	"strings"
+)
+
+const (
+	DontInheritTagPrefix = "^"
```

Comment:
So we're prefixing the tags we don't want to inherit? I thought we might have went the other way around?

Reply:
Correct. And, yes, I do want to reverse it :)

Reply:
Made the change so that tags do not inherit by default. Re-tested all the models and made fixes where appropriate.